infrastructure/sentry-subgraph/schema/events/Referee.graphql

```diff
@@ -76,10 +77,16 @@ type RefereeConfig @entity(immutable: false) {
 
 type PoolSubmission @entity(immutable: false) {
   id: String!
-  challengeId: BigInt!
+  challengeId: BigInt! @index
+  poolAddress: Bytes! @index
+  challenge: Challenge!
+  poolInfo: PoolInfo!
   stakedKeyCount: BigInt!
   winningKeyCount: BigInt!
   claimedRewardsAmount: BigInt!
-  poolAddress: Bytes!
+  createdTimestamp: BigInt!
+  createdTxHash: Bytes!
+  claimTimestamp: BigInt!
+  claimTxHash: Bytes!
   claimed: Boolean!
```

Comment:
Should any additional fields have @index param?

claimed?

Reply:
Good catch, some got added in the follow up PR https://github.com/xai-foundation/sentry/pull/281 but I will extend the claimed field there too. Thanks !